crates/bevy_reflect/src/func/args/from_arg.rs

```diff
@@ -9,15 +9,18 @@ use crate::func::args::{Arg, ArgError, ArgInfo};
 ///
 /// [derive macro]: derive@crate::Reflect
 pub trait FromArg {
```

Comment:
I would like if the documentation somewhere indicated that the purpose of `FromArg` is largely to convert functions with normal parameters into functions that take an `ArgList` as input (e.g. with `IntoFunction`)

Reply:
You’re right it should mention that. I'll need to check the other `IntoFunction`-relevant traits too